content/en/docs/multi-center/modules/hosp/d_labitems.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "d_labitems"
+linktitle: "d_labitems"
+weight: 4
+date: 2023-09-18
+description: >
+  Dimension table for *labevents* provides a description of all lab items.
+---
+
+## *d_labitems*
+
+*d_labitems* contains definitions for all `itemid` associated with lab measurements in the MIMIC database. All data in *labevents* link to the *d_labitems* table. Each unique (`fluid`, `category`, `label`) tuple in the hospital database was assigned an `itemid` in this table, and the use of this `itemid` facilitates efficient storage and querying of the data.
+
+Laboratory data contains information collected and recorded in the hospital laboratory database. This includes measurements made in wards within the hospital and clinics outside the hospital. Most concepts in this table have been mapped to LOINC codes, an openly available ontology which facilitates interoperability.
+
+For the data sourced from NMHC, Illinois law defines certain categories of information as Sensitive Protected Health Information (SPHI) which require special treatment. SPHI includes genetic counseling but does not include genetic testing.
+
+To facilitate further multi-center initiatives, the lab mappings to standard terminologies (LOINC)  will be released.
+
+### Links to
+
+* *labevents* on `itemid`
+
+## Table columns
+
+Name | Postgres data type
+---- | ----
+`itemid` | INTEGER
+`label` | VARCHAR(50)
+`fluid` | VARCHAR(50)
+`category` | VARCHAR(50)
+
+## Detailed Description
+
+### `itemid`
+
+A unique identifier for a laboratory concept. `itemid` is unique to each row, and can be used to identify data in labevents associated with a specific concept.
+
+### `label`
+
+The `label` column describes the concept which is represented by the `itemid`.
+
+We provide a list of common COVID-19 tests and measurements in the database, as defined by LOINC terminology, below:
+
+- SARS-CoV-2 (COVID-19) [Presence] in Specimen by Organism specific culture
```

Comment:
would be useful to add `itemid` and the LOINC for these examples

Reply:
updated!

Reply:
hmm I don't see it? something wrong with my viewer or did the commit not go through?

Reply:
it is actually part of commit obf0574.